contracts/scripts/native_solc_compile_all_ccip

```diff
@@ -9,6 +9,7 @@ echo " └───────────────────────
 SOLC_VERSION="0.8.24"
 OPTIMIZE_RUNS=26000
 OPTIMIZE_RUNS_OFFRAMP=500
+ZKSOLC_VERSION="1.5.6"
```

Comment:
I believe at least for CCIP this needs to be 1.5.3
this is going to be interesting now. that how the other products will handle this. I think we may need to sync with @friedemannf & a few others to nail this down once & for all

Reply:
this file is specific to ccip. there are separate files per product where you can specify the zksolc version.
i have changed this to 1.5.3